apps/dash-phylogeny/app.py

```diff
@@ -0,0 +1,811 @@
+# -*- coding: utf-8 -*-
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+import plotly.graph_objs as go
```

Comment:
Is this import used anywhere else in the app?

Reply:
Apparently no, removed